intro-js.md

```diff
@@ -0,0 +1,149 @@
+# Introduction
+Welcome to the weird and wonderful world of JavaScript development. 
+JavaScript, as a language, is used primarily in the development of web applications and websites. 
+
+## Why is JavaScript weird? 
+
+1. Lack of types
+
+JavaScript is a *loosely typed* or *dynamically typed* language
+
+```javascript
+var a = 42; //a is a number here
+var a = 'random' //a is a string here
+var a = true //a is a boolean here
+```
+
```

Comment:
Add something like `JavaScript determines types during "runtime", in other words, the type is determined when the code is running`